### PR TITLE
fix(lexical-playground): draggable blockplugin uses unexposed apis

### LINKED ISSUE #3100: Bug: `DraggableBlockPlugin` uses unexposed apis
Getting `Failed to resolve import "lexical/LexicalUtils"` when trying to use `DraggableBlockPlugin` from the playground. `getElementByKeyOrThrow` is exported by `lexical/LexicalUtils` but the npm package only contains `Lexical.{dev,prod}.js` and `getElementByKeyOrThrow` is not exported anywhere else.

Lexical version: 0.5

image

### PR DESCRIPTION
**Pull request by LuciNyan**
Closes #3100

### CODE REVIEW

File: packages/lexical-playground/src/plugins/DraggableBlockPlugin/index.tsx [hunk 1]
```diff
@@ -18,7 +18,6 @@ import {
   DROP_COMMAND,
   LexicalEditor,
 } from 'lexical';
-import {getElementByKeyOrThrow} from 'lexical/src/LexicalUtils';
```
Comment (acywatson) [inline packages/lexical-playground/src/plugins/DraggableBlockPlugin/index.tsx:21]: We could also just export it, I guess?

Comment (LuciNyan) [reply packages/lexical-playground/src/plugins/DraggableBlockPlugin/index.tsx:21]: I noticed that in other plugins, editor.getElementByKey is used when you need to get the element by key. whether we export getElementByKeyOrThrow or not, it looks better to use editor.getElementByKey consistently here. What do you think?

Comment (acywatson) [reply packages/lexical-playground/src/plugins/DraggableBlockPlugin/index.tsx:21]: I don't have a strong preference - it's mostly just a convenience method, but now you've gone from throwing an error in the case of a key lookup miss to just silently failing. If that's the behavior you want, then this is fine.

**Review by acywatson — APPROVED** [review]

File: packages/lexical-playground/src/plugins/DraggableBlockPlugin/index.tsx (hunk 1 above)
Comment (LuciNyan) [reply packages/lexical-playground/src/plugins/DraggableBlockPlugin/index.tsx:21]: Thank you for the detailed explanation here. <3

**Pull request merged**